srcpkgs/nextdns/files/nextdns_conf

```diff
@@ -0,0 +1,5 @@
+# Configuration for NextDNS
+
+# NextDNS custom configuration id
+# change id with your NextDNS id
+config id
```

Comment:
Can this file be extracted from the download tarball?

Reply:
We can't.

Reply:
Hi,
this file doesn't exist in tarball.
Nextdns doesn't work with Void out of the box because it doesn't support runit system, yet. Nextdns has commands:

```
Usage: nextdns <command> [arguments]

The commands are:

    install         install service on the system
    uninstall       uninstall service from the system
    start           start installed service
    stop            stop installed service
    restart         restart installed service
    status          return service status
    log             show service logs
    run             run the daemon
    config          manage configuration
    activate        setup the system to use NextDNS as a resolver
    deactivate      restore the resolver configuration
    version         show current version

```
That interact directly with `systemd` or `upstart` or `sysv`, etc, but not `runit`

But it's easy to configure it to work with Void, user only have to configure config file manually and run the only NextDNS commands that work: `nextdns run` so you can start it with something similar:

```
sudo nextdns run \
  -config <your config id> ...
```
In order to use it with Void as system service, you have to put your configuration in
`/etc/nextdns/nextdns_conf file` and symlink `/etc/sv/nextdnsd` to `/var/service`

To setup the system to use NextDNS as a dns resolver, un-comment the last line in `/etc/resolvconf.conf` to `name_servers=127.0.0.1` and restart resolvconf with `resolvconf -u` now NextDNS should be used as dns resolver